src/styles/globals.css

```diff
@@ -6,8 +6,7 @@
 }
 
 a {
-  color: #5CFF00;
-  /* text-decoration: none; */
+  color: #7cff33;
```

Comment:
hmm, let's keep the links color: #5CFF00 since the text in the how to play page are color: #7cff33